src/css/components/InputGroup.scss

```diff
@@ -45,22 +46,40 @@ $form-input-height: $su-xlarge !default;
 
 .InputGroup--leftIcon {
   .Input {
-    padding-left: scaleGrid(5);
+    padding-left: scaleGrid(4);
   }
 
-  .InputGroup-icon {
+  .icon-Left {
```

Comment:
Since this is a modifier of  `.InputGroup-icon`,  I would probably do something like `.InputGroup-icon--left` and same for the others: `.InputGroup-icon--right` and `.InputGroup-icon--aux`.